hfinger/hfinger_core.py

```diff
@@ -191,6 +191,47 @@ def get_content_type(hdr):
     return hdr_coded + ",".join(ret)
 
 
+def get_cache_control_value(hdr):
+    hdr_value_table = CACHECONT
+    val = hdr.split(":")[1]
+    if val[0] == " ":
+        val = val[1:]
+    hdr_coded = HDRL["cache-control"] + ":"
+    ret = []
+    if "," in val:
+        # simple splitting of compound values
+        t = []
+        if ", " in val:
```

Comment:
Maybe it would be safer to always split by `,` and then strip the resulting parts?